PCSX2/pcsx2
# USB, PAD, QT: Copy usb input

### Description of Changes
Change the copy input profile logic to also copy usb configurations.

### Rationale behind Changes
Fix for #8027. Allows the copy input profile to behave as expected and copy over all of the current input settings instead of just the controller settings.

### Suggested Testing Steps
- Open the controller settings.
- Configure a USB device with custom bindings and settings (e.g. guncon2)
- Click the create new profile button
- Give the profile a name
- When asked if you want to copy the settings from the current profile select yes
- Inspect the new input profile and make sure that the bindings and settings for all USB devices match those of the previous profile
